Makefile

```diff
@@ -199,30 +199,30 @@ i18n-django-compilemessages:
 	cd kolibri && PYTHONPATH="..:$$PYTHONPATH" python -m kolibri manage compilemessages
 
 i18n-upload: i18n-extract
-	python build_tools/i18n/crowdin.py upload-sources ${branch}
+	python packages/kolibri-tools/lib/i18n/crowdin.py upload-sources ${branch}
```

Comment:
We might want to deprecate these makefile commands, and instead document using the crowdin.py script directly. That would help avoid duplication that we're seeing in Studio and would begin to see other places:

https://github.com/learningequality/studio/pull/2520/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R73-R126

Having these aliases in the makefile was something of a historic artifact and I don't think there's a good reason to have them here